__tests__/fixtures/model-files/node_modules/@stackbit/components/presets/model1.json

```diff
@@ -0,0 +1,12 @@
+{
+    "model": "model_1",
+    "presets": [
+        {
+            "label": "ext - my preset 1",
+            "thumbnail": "path/to/ext-preset-thumbnail1.png",
+            "data": {
+                "enum_field": "thumbnail_1"
+            }
+        }
+    ]
```

Comment:
suggest adding another test that resolved absolutes path inside component library like we do with regular thumbnails:

test:
https://github.com/stackbit/stackbit-sdk/blob/58694ea6136d8fdf8e3c2f8a27fa9e4864157f8e/__tests__/fixtures/model-files/node_modules/%40stackbit/components/models/model_components_5.yaml#L13

result:
https://github.com/stackbit/stackbit-sdk/blob/58694ea6136d8fdf8e3c2f8a27fa9e4864157f8e/__tests__/config-loader.js#L179-L182

The function that maps thumbnails
https://github.com/stackbit/stackbit-sdk/blob/58694ea6136d8fdf8e3c2f8a27fa9e4864157f8e/src/config/config-loader.ts#L398-L408